api/tests/routes/backoffice/autocomplete_test.py

```diff
@@ -44,13 +44,17 @@ class AutocompleteOffererTest:
             ("ciné", {"Le Cinéma (123456789)", "Cinéma concurrent (561234789)"}),
             ("cinema", {"Le Cinéma (123456789)", "Cinéma concurrent (561234789)"}),
             ("ciné théâtre", set()),
+            ("666666", set()),
+            ("666666001", {"Le Théâtre (100200300)"}),
+            ("666666004", {"Cinéma concurrent (561234789)"}),
         ],
     )
     def test_autocomplete_offerers(self, authenticated_client, search_query, expected_texts):
-        offerers_factories.OffererFactory(siren="100200300", name="Le Théâtre")
-        offerers_factories.OffererFactory(siren="123456789", name="Le Cinéma")
-        offerers_factories.OffererFactory(siren="123444556", name="La Librairie")
-        offerers_factories.OffererFactory(siren="561234789", name="Cinéma concurrent")
+        offerers_factories.OffererFactory(id=1, siren="100200000", name="La scène")
+        offerers_factories.OffererFactory(id=666666001, siren="100200300", name="Le Théâtre")
+        offerers_factories.OffererFactory(id=666666002, siren="123456789", name="Le Cinéma")
+        offerers_factories.OffererFactory(id=666666003, siren="123444556", name="La Librairie")
+        offerers_factories.OffererFactory(id=666666004, siren="561234789", name="Cinéma concurrent")
```

Comment:
Pour tester à fond, j'aurais bien mis l'un avec un id qui soit le début du SIREN d'un autre, par exemple : 
```suggestion
        offerers_factories.OffererFactory(id=12344, siren="561234789", name="Cinéma concurrent")
```
Ainsi le dernier cas de test devient : 
```
("12344", {"Cinéma concurrent (561234789)", "La Librairie (666666003)"}),
```
et le `or_` est bien validé :)

Reply:
C'est changé

Reply:
Merci :)
Si la CI ne rapporte pas d'erreur, tu peux merger